src/clang.rs

```diff
@@ -1687,6 +1687,7 @@ impl Drop for Diagnostic {
 }
 
 /// A file which has not been saved to disk.
+#[derive(Clone)]
```

Comment:
This is not safe. `UnsavedFile` keeps pointers to its inner `CString`s so at the very least the clone implementation needs to be more subtle.